File: Brute-Force/2016. Maximum Difference Between Increasing Elements/maximumDifference.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var maximumDifference = function(nums) {
    let output = Number.NEGATIVE_INFINITY;
    
    for (let i = 0; i < nums.length; i++) {
        for (let j = i + 1; j < nums.length; j++) {
            if (nums[i] < nums[j]) {
                output = Math.max(output, nums[j] - nums[i]);
            }
        }
    }
    
    return output === Number.NEGATIVE_INFINITY ? -1 : output;
};

File: Brute-Force/2016. Maximum Difference Between Increasing Elements/maximumDifference.py
class Solution:
    def maximumDifference(self, nums: List[int]) -> int:
        output = float('-inf')
        
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if nums[i] < nums[j]:
                    output = max(output, nums[j] - nums[i])
        
        return output if output != float('-inf') else -1